bip-0009/assignments.mediawiki

```diff
@@ -0,0 +1,37 @@
+==Deployments==
+
+List of proposed deployments.
+
+State can be defined, active, failed. Dates are in UTC.
+
+{| class="wikitable"
```

Comment:
afaic, none of the fancy stuff works on Github rendering. Look at the README.mediawiki for example - it's got lots of formatting and colours, but doesnt render.